src/views/account/index.tsx

```diff
@@ -1,17 +1,72 @@
 
-import { FC } from "react";
+import { FC, useEffect, useState } from "react";
 import { UserDetails } from "../../components/UserDetails";
+import { HasAccount} from "services/UserService";
+import HasAccountRequest from "models/HasAccountRequest";
+import { useWallet } from "@solana/wallet-adapter-react";
+import { GetLocalUserStorage } from "services/UserService";
+import IUser from "models/IUser";
+
+
+
 
 export const AccountView: FC = ({ }) => {
 
```

Comment:
I think in general as this grows I'm going to have to figure out a better sln to state managment, but that's a future problem for the time being